react/week3/Buttons.js

```diff
@@ -0,0 +1,5 @@
+export default function Button({name, onClick, type}){
+    return(
+        <button onClick={onClick} type={type}>{name}</button>
```

Comment:
if you choose to have type as props, then you should  define type while using <Button> component